main.go

```diff
@@ -378,3 +388,18 @@ func WriteSubnetFile(path string, nw ip.IP4Net, ipMasq bool, bn backend.Network)
 	return os.Rename(tempFile, path)
 	//TODO - is this safe? What if it's not on the same FS?
 }
+
+func mustRunHealthz() {
+	address := net.JoinHostPort(opts.healthzIP, strconv.Itoa(opts.healthzPort))
+	log.Infof("Start healthz server on %s", address)
+
+	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte("flanneld is running"))
+	})
+
+	if err := http.ListenAndServe(address, nil); err != nil {
+		log.Errorf("Start healthz server error. %v", err)
+		panic(err)
```

Comment:
Nope. `panic` because this method is called `mustRunHealthz`. According to golang convention, we should panic when we can not do some thing that must be done.

The reason for why we use `mustRunHealthz` is because is think we should must start the health check server once we enable it. If not, we should panic or stop flannel to give user more clear result. 

@tomdee WDYT.